cppcache/src/DistributedSystemImpl.cpp

```diff
@@ -38,6 +38,10 @@
 #include "util/Log.hpp"
 #include "version.h"
 
+namespace {
+std::map<std::thread::id, std::string> m_threadNames;
```

Comment:
Does not conform to naming convention.

Reply:
I'm using this anonymous namespace to solve an apparent linking issue. Without it, this map is an unresolved reference in the clicache. Do you know of another way to solve that?

Reply:
I am referring to the variable name. This is a global variable, now a member variable and even member variables aren't supported to have the old `m_` prefix.

Reply:
Good catch.